src/PHPMailer.php

```diff
@@ -4271,6 +4307,35 @@ public static function isValidHost($host)
         return filter_var('http://' . $host, FILTER_VALIDATE_URL) !== false;
     }
 
+    /**
+     * Check whether the supplied address uses unicode in the localpart.
+     *
+     * @return bool
+     */
+    public function addressHasUnicodeLocalpart($address)
```

Comment:
I'd prefer these new methods to be `protected` - it's not necessary to make them part of the public API as they are only intended for internal use.

Reply:
Will do. Do you prefer force-push or another commit?

Reply:
I'm fine with force push for PRs.